client/webserver/locales/en-us.go

```diff
@@ -244,4 +244,12 @@ var EnUS = map[string]string{
 	"export_wallet_msg":           "Below are the seeds needed to restore your wallet in some popular external wallets. DO NOT make transactions with your external wallet while you have active trades running on the DEX.",
 	"clipboard_warning":           "Copy/Pasting a wallet seed is a potential security risk. Do this at your own risk.",
 	"fiat_exchange_rate_sources":  "Fiat Exchange Rate Sources",
+	"estimated_transaction_fee":   "Estimated Transaction Fee",
```

Comment:
maybe to make UI slightly leaner: `Estimated Transaction Fee` -> `Estimated Fee`